composer.json

```diff
@@ -8,14 +8,18 @@
         {
             "name": "Toon Verwerft",
             "email": "toon.verwerft@phpro.be"
+        },
+        {
+            "name": "Jelle Deneweth",
+            "email": "jelle.deneweth@gmail.com"
         }
     ],
     "require": {
         "php": "^8.0",
         "ext-json": "*",
         "amphp/amp": "^2.5",
         "azjezz/psl": "~1.9.1",
-        "league/uri": "^6.3",
+        "league/uri": ">= 6.3 <=6.7",
```

Comment:
remove limitation

Reply:
6.8 requires PHP8.1

Reply:
That shouldn't matter: Composer is smart enough to figure that out.